.github/workflows/tests.yml

```diff
@@ -54,17 +53,32 @@ jobs:
       test-bot-formulae-args: ${{ steps.check-labels.outputs.test-bot-formulae-args }}
       test-bot-dependents-args: ${{ steps.check-labels.outputs.test-bot-dependents-args }}
     steps:
-      - name: Show GraphQL Query Response Headers
-        if: ${{ github.event_name == 'pull_request' }}
+      - name: Count 'CI-long-timeout' labels
+        id: count_long_timeout_prs
         run: |
-          QUERY='query{ repository(owner:\"Homebrew\", name:\"homebrew-core\") { pullRequests(last: 100, states: OPEN, labels: [\"CI-long-timeout\"]) { totalCount } } }'
-          curl \
-            --include \
-            --header 'Content-Type: application/json' \
-            --header 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
-            --request POST \
-            --data "{ \"query\": \"$QUERY\" }" \
-            https://api.github.com/graphql
+          QUERY='
+            query($owner:String!, $name:String!, $label:String!) {
+              repository(owner: $owner, name: $name) {
+                pullRequests(last: 100, states: OPEN, labels: [$label]) {
+                  totalCount
+                }
+              }
+            }
+          '
+
+          OWNER='${{ github.repository_owner }}'
+          REPO_NAME=$(echo '${{ github.repository }}' | cut -f 2 -d '/')
```

Comment:
We could use `github.event.repository.name` here (see https://stackoverflow.com/a/62805013), but I couldn't find any information on it so I'd rather not rely on that.